src/schrodinger/sketcher/molviewer/atom_item.cpp

```diff
@@ -484,6 +488,50 @@ void AtomItem::updateMappingLabel()
     }
 }
 
+QString AtomItem::getTooltip() const
```

Comment:
maybe we have already agreed on a convention and I forgot, in which case apologies... but to me `getXXX` suggests return a value and pairs with set, while something like `computeXXX` suggests that operations have to be run to calculate the value... does that make sense to you?